about.html

```diff
@@ -0,0 +1,127 @@
+                        <a href="#" class="menu-link">News</a>
+                        <a href="#" class="menu-link">CC Campain</a>
+                    </div>
+                    
+                </div>
+            </nav>
+            <div class="menu_text">
+                <div class="background_text">
+                    <h2 class="principal_title">About our session</h2>
+                </div>
+                <div class="menu_paragraph">
+                    <p>Contemporary topics such as Privacy, Brand Protection and Managing Digital Investigations, Embracing Leadership That Will Thrive in the Digital Era, Cybersecurity will be top on the agenda. Interactive On-screen presentation with visual aids, Panel discussions will equip professionals with strategies and techniques for rethinking the value of workspaces, workforces, and the work.</p>
+                </div>
+                <div class="date_menu">
+                    <p>Please reach out to us throught the website for the information. Cyber security Webinar 2022! <span>cysecurity2022@gmail.com</span></p>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <!-- About section -->
+
+    <section class="about_section">
+        <h2 class="about_title">Cyber Security</h2>
+        <div class="border-width"></div>
+        <p class="about_text">The Chapter will also hold a free pre-conference virtual event on 22nd September 2021 under the theme, Cybersecurity practical Lab Learning. This all-inclusive University students group engagement is aimed at introducing them to hands-on skills sessions relating to Cybersecurity.</p>
+        <img src="./Images/wolfPictures.png" alt="logo" class="about_img">
+    </section>
+
+    <!-- Event section -->
+
+    <section class="past_event_section">
+        <h2 class="event_title">
+            Past Session
+        </h2>
+        <div class="border-width"></div>
+        <p class="event_text">Take a look at the past sessions we perfomed and the changement behind our session</p>
+      <div class="flex_event">
+        <div class="event_img">
+            <div class="background_event">
+                <img src="https://image.cnbcfm.com/api/v1/image/106906406-1625260875644-gettyimages-1199270450-syz_2991.jpeg?v=1637267361" alt="event">
+                <div class="div_event"></div>
+                <p class="event_par">2020</p>
+                <p class="event_p">Live on Zoom</p>
+            </div>
+        </div>
+
+        <div class="event_img">
+            <div class="background_event">
+                <img src="https://image.cnbcfm.com/api/v1/image/106906406-1625260875644-gettyimages-1199270450-syz_2991.jpeg?v=1637267361" alt="event">
+                <div class="div_event"></div>
+                <p class="event_par">2021</p>
+                <p class="event_p">Live on Zoom</p>
+            </div>
+        </div>
+     </div>  
+    </section>
+
+    <!-- Footer -->
+
+    <footer class="footer_home">
```

Comment:
Kindly add the partners on the About page for the mobile version:
Provided design:
![image](https://user-images.githubusercontent.com/83596135/163605540-ab04c270-0453-496b-9611-c8830c2e72bd.png)